### ISSUE TITLE
Improve "Integration with DRF " section in the documentation.

### ISSUE THREAD

**Issue by YRFT** [carltongibson/django-filter#1350]
In https://django-filter.readthedocs.io/en/stable/ref/filterset.html#declaring-filterable-fields, it says:

> Note that it is not necessary to included declared filters in a fields list - doing so will have no effect ...

But in the example of section https://django-filter.readthedocs.io/en/stable/guide/rest_framework.html#adding-a-filterset-with-filterset-class:

```python
class ProductFilter(filters.FilterSet):
    min_price = filters.NumberFilter(field_name="price", lookup_expr='gte')
    max_price = filters.NumberFilter(field_name="price", lookup_expr='lte')

    class Meta:
        model = Product
        fields = ['category', 'in_stock', 'min_price', 'max_price']
```

The `min_price` and `max_price` fields in the `Meta.fields` are unnecessary, but the documentation does not point that out. So it makes a little confusing.

Maybe it's better to delete `min_price` and `max_price` in the example.

I've made a pull request, [#1349](https://github.com/carltongibson/django-filter/pull/1349).

**Comment by timnyborg** [comment]
> Note that it is not necessary to included declared filters in a fields list - doing so will have no effect ...

But it does have an effect: form ordering!  If you omit declared filters from the field list, they appear in a FilterSet's form after automatic filters, in the order they're defined.  if included, you have control over their order declaring every single filter and restructuring your code.

Not applicable to DRF, of course, so the documentation needs two fixes.

**Comment by YRFT** [comment]
Thank you for pointing that out.

**Comment by carltongibson** [comment]
Closed in #1349
